http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/HttpServletWithSessionListenerIT.java

```diff
@@ -51,6 +63,27 @@ public void sessionSecured() {
         thenCheckActiveSessionsEqualTo(1);
     }
 
+    @Test
+    @DisplayName("Undertow servlet test http2")
```

Comment:
It would be nice if you addressed my comments from the other PR here, considering it is same code and I've already suggested changes. Mainly `underTow` is not correct.

Reply:
I don't get it at all, I removed unnecessary test code related to Undertow in my other PR (https://github.com/quarkus-qe/quarkus-test-suite/pull/1655) and put it in this new PR.